siriuspy/siriuspy/csdevice/timesys.py

```diff
@@ -469,9 +475,9 @@ def get_evg_database(prefix=None, only_evg=False):
     if only_evg:
         return db
 
-    for clc in clocks_ll2hl_map.keys():
+    for clc in Const.ClkLL2LLMap.keys():
```

Comment:
Is this name (Const.ClkLL2LLMap) right? Just checking, because there is a difference between the old and the new names meaning.

Reply:
Thanks @anacso17! It was wrong. The name was supposed to be `ClkLL2HLMap`.